spatialdata/_core/_spatialdata.py

```diff
@@ -324,17 +326,22 @@ def _write_transformations_to_disk(self, element: SpatialElement) -> None:
             else:
                 raise ValueError("Unknown element type")
 
-    def filter_by_coordinate_system(self, coordinate_system: str) -> SpatialData:
+    def filter_by_coordinate_system(
+        self, coordinate_system: Union[str, list[str]], filter_table: bool = False
+    ) -> SpatialData:
         """
-        Filter the SpatialData by a coordinate system.
+        Filter the SpatialData by one (or a list of) coordinate system.
 
-        This returns a SpatialData object with the elements containing
-        a transformation mapping to the specified coordinate system.
+        This returns a SpatialData object with the elements containing a transformation mapping to the specified
+        coordinate system(s).
 
         Parameters
         ----------
         coordinate_system
-            The coordinate system to filter by.
+            The coordinate system(s) to filter by.
+        filter_table
+            If True, the table will be filtered to only contain regions of an element belonging to the specified
+            coordinate system(s).
```

Comment:
I think this should be True by default. I understand why `False` but I don't think it should be suggested behaviour.

Reply:
changed the default to true